logstage/logstage-api/src/main/scala/izumi/logstage/macros/LogMessageMacro.scala

```diff
@@ -154,13 +156,18 @@ object LogMessageMacro {
         val sc = q"""_root_.scala.StringContext($stringContext.parts.map(_.stripMargin): _*)"""
         reifyContext(c)(sc, namedArgs)
 
-       // support .stripMargin in scala 2.13
+      // support .stripMargin in scala 2.13
       case Select(Apply(_, List(Typed(tree, _))), TermName("stripMargin")) =>
-        processExpr(c)(tree)
+        tree match {
+          case PlusExtractor(lst) =>
```

Comment:
@VladPodilnyk 
Hmm, I'm not sure this will always match (what if there are no interpolations? e.g. `"abc".stripMargin`, there probably won't be any pluses)
IMHO, using tailrec here is fine, and I don't see any performance issues - `@tailrec` recursion is essentially free, there isn't even a method call - you may move `isMultiline` argument up to `processExpr` and just recurse againt with `processExpr(c)(tree, true)`

Reply:
Done. @neko-kai, I'll merge this?

Reply:
@VladPodilnyk Yep 👍

Reply:
@VladPodilnyk Actually, could you add a test for single-line string with stripMargin & no-interpolations string with stripMargin, then merge? Thanks!